extensions/typescript-language-features/package.json

```diff
@@ -1630,7 +1630,7 @@
         },
         {
           "command": "typescript.selectTypeScriptVersion",
-          "when": "typescript.isManagedFile"
+          "when": "typescript.isManagedFile || resourceFilename == 'tsconfig.json'"
```

Comment:
I think we should fix `"typescript.isManagedFile` if there is a bug. There's already code that should enable this for config files:

https://github.com/microsoft/vscode/blob/737074e1b73876310a3bc9db0a8cf66882bacfec/extensions/typescript-language-features/src/ui/managedFileContext.ts#L54

If there's some case where this still isn't working let's try to fix that